sktime/utils/validation/series.py

```diff
@@ -62,6 +62,9 @@ def check_series(
     else:
         valid_data_types = VALID_DATA_TYPES
 
+    if isinstance(Z, np.ndarray):
+        Z = pd.Series(Z)
```

Comment:
@thayeylolu I believe that Z can also be multivariate series, represented as a `pd.DataFrame` or 2-dimensional array, so if it is a 2-dimensional array, we need to cast it into a `pd.DataFrame` rather than a `pd.Series`

Reply:
oh! I would return a dataframe in that case. Thank you Markus